server/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -249,6 +249,38 @@ public void testFillsAttributes() throws IOException {
         assertEquals("<ALPHANUM>", tokens.get(3).getType());
     }
 
+    public void testMultiPhraseRequest() throws IOException {
+        AnalyzeAction.Request request = new AnalyzeAction.Request();
+        request.analyzer("standard");
```

Comment:
nit: we could maybe add coverage for other analyzers as well (e.g. `simple`)

Reply:
I updated the test based on the pr comments